pallets/liquidity-pools-gateway/src/weights.rs

```diff
@@ -21,28 +21,80 @@ pub trait WeightInfo {
+
 impl WeightInfo for () {
 	fn set_domain_router() -> Weight {
-		Weight::from_ref_time(10_000_000)
+		// TODO: BENCHMARK CORRECTLY
+		//
+		// NOTE: Reasonable weight taken from `PoolSystem::set_max_reserve`
+		//       This one has one read and one write for sure and possible one
+		//       read for `AdminOrigin`
+		Weight::from_parts(17_000_000, 5991)
+			.saturating_add(RocksDbWeight::get().reads(2))
+			.saturating_add(RocksDbWeight::get().writes(1))
 	}
 
 	fn add_instance() -> Weight {
-		Weight::from_ref_time(10_000_000)
+		// TODO: BENCHMARK CORRECTLY
+		//
+		// NOTE: Reasonable weight taken from `PoolSystem::set_max_reserve`
+		//       This one has one read and one write for sure and possible one
+		//       read for `AdminOrigin`
+		Weight::from_parts(17_000_000, 5991)
+			.saturating_add(RocksDbWeight::get().reads(2))
+			.saturating_add(RocksDbWeight::get().writes(1))
 	}
 
 	fn remove_instance() -> Weight {
-		Weight::from_ref_time(10_000_000)
+		// TODO: BENCHMARK CORRECTLY
+		//
+		// NOTE: Reasonable weight taken from `PoolSystem::set_max_reserve`
+		//       This one has one read and one write for sure and possible one
+		//       read for `AdminOrigin`
+		Weight::from_parts(17_000_000, 5991)
+			.saturating_add(RocksDbWeight::get().reads(2))
+			.saturating_add(RocksDbWeight::get().writes(1))
 	}
 
 	fn add_relayer() -> Weight {
-		Weight::from_ref_time(10_000_000)
+		// TODO: BENCHMARK CORRECTLY
+		//
+		// NOTE: Reasonable weight taken from `PoolSystem::set_max_reserve`
+		//       This one has one read and one write for sure and possible one
+		//       read for `AdminOrigin`
+		Weight::from_parts(17_000_000, 5991)
+			.saturating_add(RocksDbWeight::get().reads(2))
+			.saturating_add(RocksDbWeight::get().writes(1))
 	}
 
 	fn remove_relayer() -> Weight {
-		Weight::from_ref_time(10_000_000)
+		// TODO: BENCHMARK CORRECTLY
+		//
+		// NOTE: Reasonable weight taken from `PoolSystem::set_max_reserve`
+		//       This one has one read and one write for sure and possible one
+		//       read for `AdminOrigin`
+		Weight::from_parts(17_000_000, 5991)
+			.saturating_add(RocksDbWeight::get().reads(2))
+			.saturating_add(RocksDbWeight::get().writes(1))
 	}
 
 	fn process_msg() -> Weight {
-		Weight::from_ref_time(10_000_000)
+		// TODO: BENCHMARK AND USE REAL WEIGHTS
+		//
+		// NOTE: For reference this weight compared to our maximum weight
+		//       * This weight      { ref_time: 4333558693, proof_size:   91070 }
+		//       * Maximum weight { ref_time: 500000000000, proof_size: 5242880 }
+		//
+		Weight::from_parts(78_019_565, 19974)
+			.saturating_add(Weight::from_ref_time(38_884_782).saturating_mul(N))
+			.saturating_add(RocksDbWeight::get().reads(8))
+			.saturating_add(RocksDbWeight::get().reads((7_u64).saturating_mul(N)))
+			.saturating_add(RocksDbWeight::get().writes(8))
+			.saturating_add(RocksDbWeight::get().writes((6_u64).saturating_mul(N)))
+			.saturating_add(Weight::from_proof_size(17774).saturating_mul(N))
```

Comment:
Note: I made sure thus default weight and PoV size is still executable. It is:
<img width="632" alt="Screenshot 2023-09-06 at 08 24 39" src="https://github.com/centrifuge/centrifuge-chain/assets/26296386/4f8d1ccf-e5c1-4f6b-b231-753d8f21a6b4">

Reply:
I did too ^^

Reply:
No, I think we should increase it.